tutorials/adding_new_layers_tensorflow.md

```diff
@@ -5,34 +5,48 @@ Follow the guide [here](adding_new_layers.md) to add your new layer to Fabrik's
 
 #### Importing a layer
 
+- Open [layers_import.py](https://github.com/Cloud-CV/Fabrik/blob/master/tensorflow_app/views/layers_import.py).
+
+    - Add a function to import your layer.
+        - Create a function name import_<layer_name> that takes one parameter, layer_ops, that is a list of all ops in the layer being imported.
+        - Get layer parameters from the operations in layer_ops and build a dictionary mapping parameter names to values.
+        - Get a list of input layers to the layer being processed using `get_input_layers(layer_ops)`.
+        - Create and return a json layer for your layer, calling `jsonLayer` with your layer_type, parameters, input layers and output_layers(optional).
```

Comment:
change this to `Create and return a json layer for new layer` replace your with new in the statement